src/test/kotlin/sirius/web/security/SamlTest.kt

```diff
@@ -56,31 +56,32 @@ class SamlTest {
+                """<?xml version="1.0"?>
+<samlp:Response xmlns:samlp="urn:oasis:names:tc:SAML:2.0:protocol" ID="5c0fd610-715f-4187-8f5e-298ca16dd37d" Version="2.0" IssueInstant="" Destination="https://memoio.test.scireum.com/saml/login" Consent="urn:oasis:names:tc:SAML:2.0:consent:unspecified" InResponseTo="identifier_1652353644351"><Issuer xmlns="urn:oasis:names:tc:SAML:2.0:assertion">https://sso.hq.scireum.com/adfs/services/trust</Issuer><samlp:Status><samlp:StatusCode Value="urn:oasis:names:tc:SAML:2.0:status:Success"/></samlp:Status><Assertion xmlns="urn:oasis:names:tc:SAML:2.0:assertion" ID="_30e5c8bb-ee3a-4290-9960-f5fae240d0e0" IssueInstant="2022-05-12T11:07:26.759Z" Version="2.0"><Issuer>https://sso.hq.scireum.com/adfs/services/trust</Issuer><ds:Signature xmlns:ds="http://www.w3.org/2000/09/xmldsig#"><ds:SignedInfo><ds:CanonicalizationMethod Algorithm="http://www.w3.org/2001/10/xml-exc-c14n#"/><ds:SignatureMethod Algorithm="http://www.w3.org/2001/04/xmldsig-more#rsa-sha256"/><ds:Reference URI="#_30e5c8bb-ee3a-4290-9960-f5fae240d0e0"><ds:Transforms><ds:Transform Algorithm="http://www.w3.org/2000/09/xmldsig#enveloped-signature"/><ds:Transform Algorithm="http://www.w3.org/2001/10/xml-exc-c14n#"/></ds:Transforms><ds:DigestMethod Algorithm="http://www.w3.org/2001/04/xmlenc#sha256"/><ds:DigestValue>oSkdX5u1Xt5PoFYL4HB+3iN9Y5Flmo8PJcL72gLBPr0=</ds:DigestValue></ds:Reference></ds:SignedInfo><ds:SignatureValue>wK618x44fMUudGg4xnGw8gETbVxMYGv73YMpGuD5gS8mOcU1xS9t2gninPexZEzh
```

Comment:
Tried to improve the display/indentation of these XML blocks but when formatting them with IntelliJ or VS Code and putting the result into the string literals the tests fail.